modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js

```diff
@@ -83,6 +83,10 @@ function toArray(value = '') {
 		newValue = [newValue];
 	}
 
+	if (newValue.some((i) => typeof i === 'number')) {
+		newValue = newValue.map((item) => item.toString());
+	}
+
```

Comment:
Hi @matyaswollner!
After some analysis, as `newValue` array will only come as strings or numbers, we can avoid the number checking and set `newValue` array values to string.
So we can just return `newValue` making the map to convert all values to string.
Like this:

```suggestion
	return newValue.map((value) => value.toString());
```

Thanks!